pkg/oc/cli/cmd/newbuild.go

```diff
@@ -152,7 +152,6 @@ func (o *NewBuildOptions) Complete(baseName, commandName string, f *clientcmd.Fa
 		return err
 	}
 
-	bo.Config.AllowNonNumericExposedPorts = true
```

Comment:
given the `AllowNonNumericExposedPorts` was used only in `newbuild.go` and `newapp.go` and in both cases would have been set to `true`, I think it is unnecessary at this point.

Reply:
this was the field I mentioned in another of your recent PRs, right ?

If I could reword you description of the behavior, based on looking at the uses, by always setting this to true in newbuild.go we are bypassing the validate port check for `oc new-build`, and by leaving it alone with newapp.go, we are always validating the ports for `oc new-app`

I think a decision needs to be made on whether that behavior should continue ... in particular why was the port check bypassed for `oc new-build` .... @bparees probably remembers ... if not, we'll ping some former team members.

We can't delete that field if we still want to always bypass the port check for `oc new-build`

Reply:
Yes, it is exactly the same variable as mentioned in https://github.com/openshift/origin/pull/18964, but there I thought the desired state was a little different. I was under an impression we wanted to keep the validation of ports for `oc new-app` and fail if there was any issue. So it made sense to keep `AllowNonNumericExposedPorts` there to differentiate between `oc new-app` and `oc new-build`.

Later our QE and @bparees corrected me that we want best effort evaluation of ports in `EXPOSE`, create only a warning if we can't evaluate some port and continue further ignoring that problematic port.

> We can't delete that field if we still want to always bypass the port check for oc new-build

This PR makes it so the port check is "bypassed" for both `oc new-build` and `oc new-app`, that is why I think it should be safe to remove it

Reply:
once upon a time i believe the behavior was:

for new-app, we failed on invalid/non-numeric ports
for new-build, since we don't have to create services, we could safely ignore invalid/non-numeric ports.

Hence why newbuild.go was setting this to true (and newapp.go would be defaulting it to false).

Since you're changing newapp to tolerate invalid ports with a warning, removing this is probably fine.